files/mygui/openmw_chargen_create_class.layout

```diff
@@ -76,6 +76,9 @@
         <!-- Dialog buttons -->
         <Widget type="HBox" position="0 158 482 24">
             <Widget type="Spacer"/>
+            <Widget type="AutoSizedButton" skin="MW_Button" name="RandomButton">
+                <Property key="Caption" value="Random"/>
```

Comment:
Non-localizable strings are not good.

Reply:
Where are openmw-specific localizable strings stored?

Reply:
Nowhere. Zini said that custom localization is only post-1.0.